__tests__/index.test.js

```diff
@@ -6,7 +6,7 @@ const add = require("../index");
 test("adds 1 + 2 to equal 3", () => {
   expect(add(1, 2)).toBe(3);
 });
-
+//
 // this might fail randomly
 test("might fail randomly", () => {
   expect(add(1, 2)).toBe(Math.random() > 0.5 ? 3 : 4);
```

Comment:
This is a very log comment.
Not expected but lets see how this affects the message to be sent on slack.
This is a very log comment.
Not expected but lets see how this affects the message to be sent on slack.
This is a very log comment.
Not expected but lets see how this affects the message to be sent on slack.